src/software/backend/backend.cpp

```diff
@@ -1,22 +1,27 @@
 #include "software/backend/backend.h"
 
+#include "software/proto/message_translation/tbots_protobuf.h"
+
 void Backend::receiveRobotStatus(TbotsProto::RobotStatus msg)
 {
     SensorProto sensor_msg;
     *(sensor_msg.add_robot_status_msgs()) = msg;
+    (*sensor_msg.mutable_time_received()) = *createCurrentTimestamp();
```

Comment:
nit: looks kinda weird, but it probably works. Same for the rest of the file
```suggestion
    *(sensor_msg.mutable_time_received()) = *createCurrentTimestamp();
```